app/editor/src/utils/getSortableOptions.ts

```diff
@@ -14,5 +15,5 @@ export const getSortableOptions = <T extends ISortableModel<any>>(
   prepend: IOptionItem[] = [],
   map: (item: T) => IOptionItem = (i) => new OptionItem(i.name, i.id),
 ) => {
-  return prepend.concat([...items].sort(sortSortable).map(map));
+  return prepend.concat([...filterEnabled(items)].sort(sortSortable).map(map));
```

Comment:
Not exactly what I was thinking, but should be fine for now.  I was thinking the `getSortableOptions` probably should be extended to have an option filter by enabled.  However, that may have undesirable outcomes too.